### PR TITLE
Add guidelines for communication types

### LINKED ISSUE #301: Add general guidelines for speaking to hackers
Include details on the following
- Speaking LOUDLY
- When to Slack
- When to announce individually
- When to announce to a group

### PR DESCRIPTION
**Pull request by zachlatta**
This change outlines the different types of communication with the hackers in Hack Camp. Closes #301.

I really tried the whole "having emotions" angle with this PR. It was kind of strange, mind you, but here it is. I apologize in advance.

### CODE REVIEW

File: cohort_4/playbook/ACTIVITIES.md [hunk 1]
```diff
@@ -756,3 +760,78 @@ the technical mentors for Hack Camp will be pinged. For example:
...
+In this modern day and age, we're fortunate to have _not one_, _**not two**_,
+**but THREE** methods of communication in camp.
+
+1. Announcing to the whole group
+2. Sharing an announcement to each hacker individually
+3. Using `@channel` on Slack
+
+_But when do we use these marvelous methods of communication, you ask?_
+
+## Announcing to the Whole Group
```
Comment (maxwofford) [inline cohort_4/playbook/ACTIVITIES.md:775]: Maybe include examples of announcements to the whole group, like on lines 832-837.

Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md:775]: Would you be interested in adding that in another PR?

Comment (maxwofford) [inline cohort_4/playbook/ACTIVITIES.md:775]: Perhaps if it brought up by a club leader.

Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md:775]: Brought up as in one of them saying it’d be valuable?

On Wednesday, August 5, 2015, Max Wofford notifications@github.com wrote:

> In cohort_4/playbook/ACTIVITIES.md
> https://github.com/hackedu/hack-camp/pull/339#discussion_r36371711:
> 
> > @@ -756,3 +760,78 @@ the technical mentors for Hack Camp will be pinged. For example:
> >  If you'd like to be added/removed from the list, let
> >  [@zachlatta](https://github.com/zachlatta) know and he'll get right on it (say
> >  `slackbot help` somewhere in the chat to see who's currently on the list).
> > +
> > +# Communicating with the Group
> > +
> > +In this modern day and age, we're fortunate to have _not one_, _**not two**_,
> > +**but THREE** methods of communication in camp.
> > +
> > +1. Announcing to the whole group
> > +2. Sharing an announcement to each hacker individually
> > +3. Using `@channel` on Slack
> > +
> > +_But when do we use these marvelous methods of communication, you ask?_
> > +
> > +## Announcing to the Whole Group
> 
> Perhaps if it brought up by a club leader.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hackedu/hack-camp/pull/339/files#r36371711.

## 

## 

Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta

Comment (maxwofford) [inline cohort_4/playbook/ACTIVITIES.md:775]: Yep!
